src/Uno.UWP/UI/Input/GestureRecognizer.cs

```diff
@@ -183,25 +183,8 @@ internal GestureSettings PreventEvents(uint pointerId, GestureSettings events)
 		{
 			if (_gestures.TryGetValue(pointerId, out var gesture))
 			{
-				if (events.HasFlag(GestureSettings.Tap))
-				{
-					gesture.PreventTap();
-				}
-
-				if (events.HasFlag(GestureSettings.RightTap))
-				{
-					gesture.PreventRightTap();
-				}
-
-				if (events.HasFlag(GestureSettings.DoubleTap))
-				{
-					gesture.PreventDoubleTap();
-				}
-
-				if (events.HasFlag(GestureSettings.Hold))
-				{
-					gesture.PreventHolding();
-				}
+				const GestureSettings PreventionMask = GestureSettings.Tap | GestureSettings.RightTap | GestureSettings.DoubleTap | GestureSettings.Hold;
+				gesture.PreventGestures(events & PreventionMask);
```

Comment:
```suggestion
				gesture.PreventGestures(events & GestureSettingsHelper.SupportedGestures);
```

Reply:
This set is different from SupportedGestures

Reply:
Yes it has the `HoldWithMouse` which is never enabled by `UIElement` (which uses this method). But basically I think that there is no reason to not allow the `Prevent<Events|Gestures>` to also disable that gesture and it would be more future proof to re-use that set than creating a specific one here (so when we will add support for the missing `CrossSlide` gesture, we will just have to review usages of the existing set).

Reply:
Great! Will make the change.